src/main/java/ca/corefacility/bioinformatics/irida/ria/web/samples/SamplesController.java

```diff
@@ -581,7 +581,10 @@ public String concatenateSequenceFiles(@PathVariable Long sampleId, @RequestPara
 		try {
 			sequencingObjectService.concatenateSequences(Lists.newArrayList(readMultiple), filename, sample,
 					removeOriginals);
-		} catch (ConcatenateException ex) {
+		} catch (IOException e) {
+			logger.error("Error reading files: ", e);
```

Comment:
This is just swallowing the exception, should it give the concatenation error and redirect like the catch below?

Reply:
Removed the IOException catch block in 56ac519f9ce8049dfaf894978357291373d24e8a as it is never thrown.